mlx/ops.cpp

```diff
@@ -696,6 +696,54 @@ split(const array& a, int num_splits, StreamOrDevice s /* = {} */) {
   return split(a, num_splits, 0, to_stream(s));
 }
 
+std::vector<array> meshgrid(
+    std::vector<array>& arrays,
+    const bool copy /* = true */,
+    const bool sparse /* = false */,
+    const std::string& indexing /* = "xy" */,
+    StreamOrDevice s /* = {} */) {
+  std::vector<std::string> valid = {"xy", "ij"};
+  if (std::find(valid.begin(), valid.end(), indexing) == valid.end()) {
```

Comment:
This should be `if (indexing != "xy" && indexing != "ij")`, no need for vector creation etc.